Detectors/ITSMFT/ITS/reconstruction/include/ITSReconstruction/CAaux.h

```diff
@@ -2,9 +2,12 @@
 #define ALIITSUCACELL_H
 
 #include <vector>
+#include <array>
 #include <cmath>
 #include "DetectorsBase/Track.h"
 #include "DetectorsBase/Constants.h"
+using std::array;
```

Comment:
Thanks! Digging around I also discovered that this can be checked also statically by clang http://clang.llvm.org/docs/DiagnosticsReference.html#wheader-hygiene